### ISSUE TITLE
reign OVER / rule OVER

### ISSUE THREAD

**Issue by esmanning** [carmls/snacs-guidelines#119]
> Kings do not own, they reign OVER.
lpp_13
sent_id = lpp_1943.707

> OVER what could this king really rule?
lpp_10
sent_id = lpp_1943.521

More fun with force dynamics! We've done ComparisonRef\~Locus based on the decision for advantage OVER (#54), but aren't happy with it.

**Comment by aryamanarora** [comment]
Hindi has "reign ON" with similar issues.

I also feel like Theme~Locus or something is more appropriate, since this isn't actually comparing directly like "advantage OVER" is.

**Comment by nschneid** [comment]
Agreed, unlike *advantage OVER* this is not expressing a comparison, it's expressing an act of reigning + a power dynamic present in that reigning. An ideal approach might be something like Theme~>DirectionOfForce. Can Direction cover direction of force?

**Comment by esmanning** [comment]
There are cases where direction of motion is also direction of physical force (The toddler kicked AT the wall), so it could make sense to include non-physical force dynamics there as well, but that doesn't seem to be covered in the current guidelines.
